README.md

```diff
@@ -1,2 +1,50 @@
 # TSC
-The AdoptOpenJDK Technical Steering Comittee 
+The AdoptOpenJDK Technical Steering Comittee is the technical governing body of the AdoptOpenJDK organisation. It admits and oversees all top-level Projects in the AdoptOpenJDK organisation.
+
+### Dependent Projects
+These projects are located in the following repositories:
+
+* https://github.com/AdoptOpenJDK/TSC
+* https://github.com/AdoptOpenJDK/openjdk-build
+* https://github.com/AdoptOpenJDK/openjdk-website
+* https://github.com/AdoptOpenJDK/openjdk-tests
+* https://github.com/AdoptOpenJDK/openjdk-api
+* https://github.com/AdoptOpenJDK/openjdk-nightly
+* https://github.com/AdoptOpenJDK/openjdk-releases
+* https://github.com/AdoptOpenJDK/openjdk-jdk8u
+* https://github.com/AdoptOpenJDK/openjdk-jdk9
+* https://github.com/AdoptOpenJDK/openjdk-infrastructure
+* https://github.com/AdoptOpenJDK/openjdk-website-backend
+* https://github.com/AdoptOpenJDK/secrets _private_
+
+### List of TSC Responsibilities
```

Comment:
While these responsibilities are good to see listed, I would hope that we are open to anyone coming to the project and contributing in these areas.  The goal should be to work in the open, and achieve consensus with all participants based on technical merit wherever possible.  I would not expect anyone to "flex their TSC credentials".